packages/manager/modules/anthos/src/dashboard/general-information/general-information.html

```diff
@@ -0,0 +1,235 @@
+                        <oui-action-menu-item data-on-click="$ctrl.onAnthosRegeneratePasswordClick()">
+                                <span
+                                    data-translate="anthos_tenant_dashboard_general_information_tile_security_center_access_anthos_menu_regenerate_pwd"
+                                ></span>
+                        </oui-action-menu-item>
+                    </oui-action-menu>
+
+                </oui-tile-definition>
+
+                <!--Netapp access-->
+                <oui-tile-definition
+                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_security_center_access_netapp_title' | translate }}">
+
+                    <oui-tile-description>
+                        <oui-clipboard class="mb-2" data-model="$ctrl.tenant.storage.accessUrl"></oui-clipboard>
+                        <oui-button
+                            data-size="m"
+                            data-icon-right="oui-icon-external-link"
+                            data-on-click="$ctrl.onOpenNetappConsoleClick()">
+                            <span data-translate="anthos_tenant_dashboard_general_information_tile_security_center_access_netapp_cta_start_console"></span>
+                        </oui-button>
+                    </oui-tile-description>
+
+                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
+                        <oui-action-menu-item data-on-click="$ctrl.onNetappRegeneratePasswordClick()">
+                                <span
+                                    data-translate="anthos_tenant_dashboard_general_information_tile_security_center_access_netapp_menu_regenerate_pwd"
+                                ></span>
+                        </oui-action-menu-item>
+                    </oui-action-menu>
+
+                </oui-tile-definition>
+
+            </oui-tile>
+        </div>
+
+        <!--Commitment tile-->
+        <div class="col-md-4">
+            <billing-subscription-tile
+                data-service-infos="$ctrl.serviceInfo"
+                data-user="$ctrl.user"
+                data-service-path="{{ ::'/dedicated/anthos/tenants/' + $ctrl.serviceName }}"
+                data-with-engagement="$ctrl.isCommitmentAvailable"
+                data-highlight-engagement="$ctrl.shouldReengage"
+                data-go-to-commit="$ctrl.goToCommit()"
+                data-go-to-cancel-commit="$ctrl.goToCancelCommit()"
+                data-go-to-cancel-resiliation="$ctrl.goToCancelResiliation()"
+                data-go-to-resiliation="$ctrl.goToResiliation()"
+                data-on-error="$ctrl.onBillingInformationError(error)"
+                data-tracking-prefix="{{ ::$ctrl.trackingPrefix }}"
+                data-commit-impression-data="$ctrl.shouldReengage ?
+                    $ctrl.RECOMMIT_IMPRESSION_TRACKING_DATA :
+                    $ctrl.COMMIT_IMPRESSION_TRACKING_DATA"
```

Comment:
I don't see these constants defined anywhere.
Also don't see the trackingPrefix?

Reply:
Currently i let them like this, but we will implement them on dedicated feature.